### ISSUE TITLE
Work down code/docs TODOs for 1.2.0

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2491]
Current TODOs as of 2021-03-07 in master.

Showing only the TODOs for pywbem code and docs that do not call out specific issues.

```
pywbem/_cim_operations.py:1631:2: W0511: TODO execute stage of wbem connection if not already executed. (fixme)
pywbem/_cim_operations.py:4444:2: W0511: TODO: Fix current impl. that assumes instance with path. (fixme)
pywbem/_cim_operations.py:6478:2: W0511: TODO: Change to yield each Open/Pull instead of first (fixme)
pywbem/_listener.py:988:2: W0511: TODO: Describe how the processing threads terminate. (fixme)
pywbem/_mof_compiler.py:1692:2: W0511: TODO: Check for invalid scopes (fixme)
pywbem/_recorder.py:577:2: W0511: TODO: future bypassed code to only ouput name and method if the (fixme)
pywbem/_server.py:143:2: W0511: TODO: Clarify which OpenPegasus versions need root/PGInterOp? (fixme)
pywbem_mock/_mainprovider.py:1007:2: W0511: TODO/KS: Future: This should route through DeleteInstance to (fixme)
pywbem_mock/_namespaceprovider.py:39:2: W0511: TODO: Should be able to import from pywbem_mock (fixme)
pywbem_mock/_providerdispatcher.py:224:2: W0511: TODO: Consolidate this deep copy with the shallow copy in (fixme)
pywbem_mock/_providerdispatcher.py:358:2: W0511: TODO: Consolidate this deep copy with the shallow copy in (fixme)
docs/concepts.rst:TODO: Write this section
docs/concepts.rst:TODO: Write this section
docs/concepts.rst:TODO: Write this section
docs/concepts.rst:TODO Write this section
docs/concepts.rst:TODO - Write this section
docs/concepts.rst:TODO: Section on indications and subscriptions
docs/concepts.rst:TODO: Create this section describing profiles, wh
```

**Comment by KSchopmeyer** [comment]
Can we consider this one closed with PR #26

**Comment by andy-maier** [comment]
At this point, there are the following TODOs left in Python code (excluding attic and tests):
```
./pywbem/_recorder.py:            # TODO: future bypassed code to only ouput name and method if the
./pywbem/_cim_operations.py:                # TODO execute stage of wbem connection if not already executed.
./pywbem/_cim_operations.py:                    # FUTURE TODO: Change to yield each Open/Pull instead of
./pywbem/_listener.py:        # TODO: Describe how the processing threads terminate.
./pywbem_mock/_wbemconnection_mock.py:        # Issue #2062: TODO/ks FUTURE Consider sorting to preserve order of

./build_moftab.py:VERBOSE = True  # TODO: Pass as command line option
./docs/conf.py:# TODO: Find out how the targeted Python version for auto-generated links
./examples/pegasusindicationtest.py:        # TODO filter for our sub mgr
./examples/send_indications.py:    # we don't TODO try None
```
